integration-testing/njs/http.js

```diff
@@ -0,0 +1,13 @@
+function echo(r) {
```

Comment:
http.js is too generic of a name.
people will just end up piling a million different tests in here.
can we name this just echo.js?

Reply:
Well, we may add more functions to this file in the future. I'd probably not add a separate file. So, maybe `server.js` ?

Reply:
whatever. we'll rename in the future if we need